src/java/org/apache/cassandra/cql3/statements/AlterViewStatement.java

```diff
@@ -62,6 +62,8 @@ public void validate(ClientState state)
             throw new InvalidRequestException("Cannot use ALTER MATERIALIZED VIEW on Table");
 
         ViewMetadata current = Schema.instance.getView(keyspace(), columnFamily());
+        if (current == null)
```

Comment:
This class looks different now. From the first glance I can see there is a null check:
```
if (null == view)
            throw ire("Materialized view '%s.%s' doesn't exist", keyspaceName, viewName);
```
Could you please rebase your changes and double check whether the issue still actual.